.github/workflows/main.yml

```diff
@@ -37,7 +37,6 @@ jobs:
           heroku_api_key: ${{secrets.HEROKU_KEY}}
           heroku_app_name: HEROKU_APP_NAME
           heroku_email: HEROKU_EMAIL
-
```

Comment:
not on this line but as we had a call reviewing this PR you should add .env description token configuration to avoid login error thanks

Reply:
Thanks a lot and the description was edit to address you concern